src/include/rados/rados_types.hpp

```diff
@@ -327,5 +327,15 @@ struct inconsistent_snapset_t {
  */
 const std::string all_nspaces(LIBRADOS_ALL_NSPACES);
 
+struct notify_ack_t {
+  uint64_t notifier_id;
+  uint64_t cookie;
+  bufferlist payload_bl;
```

Comment:
```suggestion
  ceph::bufferlist payload_bl;
```

otherwise it breaks the build of rocksdb's librados env. see https://pulpito.ceph.com/kchai-2020-09-16_14:28:48-rados-wip-kefu-testing-2020-09-16-1741-distro-basic-smithi/5440252/